content/getting-started/yaml.md

```diff
@@ -307,3 +302,69 @@ scripts:
     echo 'This is a Post-publish script'
     echo 'This script is multiline'
 ```
+
+## Conditional execution
```

Comment:
Let's add an intro:
You can skip building particular commits or watch for changes in specific files to trigger builds. It's possible to define build conditions per workflow or specific build steps.